src/index.js

```diff
@@ -138,12 +138,11 @@ module.exports = class Bitwap {
       cb(err, block)
     }
 
-    this.getBlocks([key], (err, res) => {
-      if (err) {
-        return done(err)
-      }
+    this.getBlocks([key], (results) => {
+      const err = results[key].error
+      const block = results[key].block
 
-      done(null, res[0])
+      done(err, block)
```

Comment:
ah, nvm, I misread and thought this was a function declaration, I was wondering that a array argument was and why only return one block in a func that is called getBlocks 

Thank you :)